pkg/controllers/webhook/controller.go

```diff
@@ -896,7 +896,13 @@ func (c *controller) mergeWebhook(dst *webhook, policy kyvernov1.PolicyInterface
 		}
 	}
 	for _, gvr := range gvrsList {
-		dst.set(gvr)
+		if policy.GetNamespace() == "" {
```

Comment:
```suggestion
		if !policy.IsNamespaced() {
```

Reply:
I tested this, but it did not work as desired at the time. That was the reason why I proceeded with `policy.GetNamespace()`. But maybe I did something else wrong.